recipes-domd/agl/files/meta-xt-prod-extra/recipes-connectivity/dnsmasq/dnsmasq_2.%.bbappend

```diff
@@ -0,0 +1,9 @@
+do_install_append() {
+    # Make dnsmasq to listen only on bridge interface
+    echo "interface=xenbr0" >> ${D}${sysconfdir}/dnsmasq.conf
+
+    # Configure addresses for DomF and DomA. Mac addresses
+    # are the same as in /xt/conf/*.conf
+    echo "dhcp-host=08:00:27:ff:cb:cd,domf,192.168.0.3,infinite" >> ${D}${sysconfdir}/dnsmasq.conf
```

Comment:
We will fail some time in the future with these hardcodes, please open an issue on this, so we can resolve it right later